public/index.html

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html lang="ko">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://developers.kakao.com/sdk/js/kakao.min.js"></script>
+    <script>
+      Kakao.init('process.env.REACT_APP_KAKAO_KEY');
```

Comment:
```suggestion
      Kakao.init(process.env.REACT_APP_KAKAO_KEY);
```
이렇게 써야 하지 않을까요? 이대로는 그냥 문자열을 전달하는 상황일거 같습니다

Reply:
앗 맞네요 ㅎㅎ 감사합니다 !!